housekeeper/store/api/handlers/update.py

```diff
@@ -59,3 +54,13 @@ def update_finished_retrieval_task(self, retrieval_task_id: int) -> None:
         ).all()
         for archive in completed_archives:
             self.update_retrieval_time_stamp(archive=archive)
+
+    @staticmethod
+    def update_retrieval_task_id(archive: Archive, retrieval_task_id: int):
```

Comment:
I assume the caller is responsible to commit the changes made to the entry then?

Reply:
Yes, seems as if the rule of thumb is that hk.py in cg commits the changes done when calling these methods in HouseKeeper.

Reply:
I don't think we have a set pattern for when and how we commit sessions